h2o-core/src/main/java/water/fvec/ByteVec.java

```diff
@@ -54,22 +54,26 @@
     }
   }
 
-  /** Open a stream view over the underlying data  */
+  /**
+   * Open a stream view over the underlying data
+   */
   public InputStream openStream(final Key job_key) {
-    return new InputStream() {
-      final long [] sz = new long[1];
+    InputStream is = new InputStream() {
+      final long[] sz = new long[1];
```

Comment:
Yeah looks like it's not used. Not sure about the padding, don't see why would you ever pad. My guess is it's dead code from the past. Not gonna do anything about it in this PR though, need to get the fix in.